### DISCUSSION TITLE
[Bug] New PR discussion comments hiding

### DISCUSSION THREAD

**Discussion by htcfreek** [community/community#5301] [Mobile]
When I open a PR discussion from a new comment notification then all comments newer than the marked/blue one (that from the notification) hides fast after showing the discussion.

**Comment by elisealix22** [comment]
Hi @htcfreek, are you using Android or iOS? If possible, is there any chance you can grab a screenshot or video to help us understand the issue? If not, is the issue that the newer comments (the ones below the blue one on the timeline) are not visible once the comment loads?

**Reply by htcfreek** [reply]
- It is Android.
- The newer comments are hiding short after everything us loaded. Then only the blue marked comment is visible.
- No I can't provide a video.

By the way: I think it would be more handy that the full pr discussion is opening when clicking on a notification that is marked as read. That will save me on step.

**Reply by elisealix22** [reply]
Thank you for the additional information! I'll add this note to our internal tracking issue. I'm having a hard time reproducing as the newer comments are rendering below the blue one in pull requests for me, but let me know if this is not what you're referring to.

![Screenshot_20210831-092336](https://user-images.githubusercontent.com/556051/131510549-b59db391-faca-45f9-bb99-98a22a1f78d6.png)
